CHANGELOG.md

```diff
@@ -0,0 +1,195 @@
+# Changelog
+
+## 2.1.0 - [#29](https://github.com/openfisca/country-template/pull/29) [#30](https://github.com/openfisca/country-template/pull/30)
```

Comment:
This Changelog is the file that tells the history of the evolution of your model.

It is very useful when you start having re-users who don't directly work with you (for exemple, someone who develops a web app on top of your model). Following your Changelog, they can know what's new in your model, and decide to upgrade, or to wait.

I'd encourage you to edit this file to add some information about what you started doing: what part of the legislation you started modeling, what reforms did you code, etc. 

The current content of this file is an example of Changelog. You can keep it somewhere as a reference, but it is not really about _your_ model, and it may confuse your re-users :).

Reply:
You're totally right, i didn't really know how to modify this file. We will try to emulate the openfisca-country-template and openfisca-france changelog file and adapt it to our code.